packages/common/src/types.ts

```diff
@@ -1156,12 +1342,19 @@ export interface BakerElectionInfo {
     lotteryPower: number;
 }
 
-export interface ElectionInfo {
-    electionDifficulty: number;
+export interface ElectionInfoCommon {
     electionNonce: HexString;
     bakerElectionInfo: BakerElectionInfo[];
 }
 
+export interface ElectionInfoV0 extends ElectionInfoCommon {
+    electionDifficulty: number;
+}
+
+export type ElectionInfoV1 = ElectionInfoCommon;
```

Comment:
Maybe we could at least add documentation to every type introduced and modified, since we have the documentation now.